packages/relay-runtime/handlers/connection/MutationHandlers.js

```diff
@@ -27,9 +27,37 @@ import type {
 const DeleteRecordHandler = {
   update: (store: RecordSourceProxy, payload: HandleFieldPayload) => {
     const record = store.get(payload.dataID);
+
     if (record != null) {
       const id = record.getValue(payload.fieldKey);
+
       if (typeof id === 'string') {
+        const {connections} = payload.handleArgs;
```

Comment:
I'm stuck on `connections` not appearing here in `handleArgs`, even though I _think_ I'm doing the transform right for passing the arguments on to here. But yeah, it's not working so obviously I'm doing something wrong.